src/itest/kotlin/nl/lifely/zac/itest/SignaleringenRestServiceTest.kt

```diff
@@ -0,0 +1,278 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.lifely.zac.itest
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.kotest.assertions.json.shouldBeJsonArray
+import io.kotest.assertions.json.shouldContainJsonKey
+import io.kotest.assertions.json.shouldContainJsonKeyValue
+import io.kotest.assertions.nondeterministic.eventually
+import io.kotest.assertions.nondeterministic.eventuallyConfig
+import io.kotest.core.spec.Order
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.date.shouldBeBetween
+import io.kotest.matchers.shouldBe
+import nl.lifely.zac.itest.client.ItestHttpClient
+import nl.lifely.zac.itest.config.ItestConfiguration
+import nl.lifely.zac.itest.config.ItestConfiguration.OPEN_ZAAK_BASE_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.OPEN_ZAAK_EXTERNAL_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.START_DATE
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_GROUP_A_ID
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_TASK_COMPLETED
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_USER_1_ID
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import okhttp3.Headers
+import org.json.JSONArray
+import org.json.JSONObject
+import org.mockserver.model.HttpStatusCode
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
+
+/**
+ * This test assumes previous tests completed successfully.
+ */
+@Order(TEST_SPEC_ORDER_AFTER_TASK_COMPLETED)
+class SignaleringenRestServiceTest : BehaviorSpec() {
+    private val logger = KotlinLogging.logger {}
+    private val itestHttpClient = ItestHttpClient()
+
+    init {
```

Comment:
I think there is no need for this `init` here?
```suggestion
class SignaleringenRestServiceTest : BehaviorSpec({
    private val logger = KotlinLogging.logger {}
    private val itestHttpClient = ItestHttpClient()
```